package.json

```diff
@@ -35,13 +35,11 @@
     "rxjs": "5.4.0",
     "ts-node": "4.0.1",
     "unique-string": "^1.0.0",
-    "zone.js": "^0.8.11"
-  },
-  "devDependencies": {
     "@angular/compiler": "4.1.3",
     "@angular/core": "4.1.3",
     "codelyzer": "3.0.1",
     "gitignore-to-glob": "0.3.0",
+    "zone.js": "^0.8.11",
```

Comment:
seems better to use strict versions of dependencies to avoid unexpected crashes which you can't handle